docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,122 @@
+* The target cluster must be on 5.3.x Enterprise Edition or the latest {hazelcast-cloud} release
+
+NOTE: {hazelcast-cloud} Trial and {hazelcast-cloud} Standard have a limit of 14GB of primal data. If you require more, you must use {hazelcast-cloud} Dedicated. For further information on the available {hazelcast-cloud} editions, refer to the https://docs.hazelcast.com/cloud/overview[Hazelcast {hazelcast-cloud}^] documentation.
+
+* You must specify at least one data structure name in the migration configuration file
+* All data structures specified in the migration configuration must exist in the source cluster
+* Any data structures that already exist on the target cluster are not migrated
+* Any empty data structures are not migrated
+
+== Migrate Your Data
+
+To migrate your data, you must complete the following steps:
+
+. xref:migrate:data-migration-tool.adoc#start-the-source-cluster[Start the source cluster]
++
+The source cluster is your existing cluster, the one that you want to migrate.
++
+NOTE: Hazelcast recommends that the source cluster is put in a `PASSIVE` state before you start the migration. For information on changing the state, see xref:maintain-cluster:cluster-member-states.adoc#changing-a-clusters-state[Changing a Cluster's State]. 
++
+If necessary, you can add data to the source cluster before continuing. For example, this can be done when testing a migration using a Development cluster. For further information on doing this, see the xref:migrate:data-migration-tool.adoc#add-data-to-cluster[Add Data to Cluster] section.
++ 
+You must also update the configuration for the source cluster and related data structures. For further information on doing this, see the xref:migrate:data-migration-tool.adoc#update-the-configuration[Update the configuration] section.
+
+. xref:migrate:data-migration-tool.adoc#check-the-target-cluster[Check the target cluster]
++
+The target cluster is the new cluster to which you want to migrate the source cluster.
+
+. xref:migrate:data-migration-tool.adoc#start-the-migration-cluster[Start the migration cluster]
++
+The cluster created by the custom Hazelcast distribution.
+
+. xref:migrate:data-migration-tool.adoc#run-the-migration[Run the Migration]
+
+. xref:migrate:data-migration-tool.adoc#verify-the-migrated-data[Verify the migrated data]
+
+NOTE: If you are using the DMT to test a migration, use a Development cluster when following the steps. 
+
+==== Limited Migration Cluster License
+
+A 10-node limited license is included for use with your migration cluster. 
 
-This guide consists of two sets of instructions; one for trying the DMT, other to use it with your existing cluster and maps.
+This license is valid for 60 days following download and can be used only for data migration and trial purposes. 
 
-== Trying Out the DMT
+If migrating to a {hazelcast-cloud} cluster, or an Enterprise Edition cluster that uses a TLS connection, security is enforced.
```

Comment:
this is a bit strange. If customer uses a EE cluster with TLS then they need to use TLS obviously. We don't enforce anything in that case. Only requirement is that viridian needs TLS and it's not related to DMT. It's true for all connections.

Reply:
an SSL connection is used?

or just remove this bullet?

Reply:
Yeah I think we should remove this sentence. There's nothing we enforce extra here regarding TLS. We only enforce target cluster being EE/Viridian.

Reply:
Removed sentence